### PR TITLE
[Effortless] add the Effortless project

### PR DESCRIPTION
**Pull request by robbkidd**
Initial cut at documenting the Effortless project and the humans involved in decision-making.

These are the members I am aware of. A reminder that order of members within a section is strictly alphabetical, not hierarchical.

### CODE REVIEW

File: projects/effortless.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Effortless
+
+The Effortless project is ...?
```
Comment (robbkidd) [inline projects/effortless.md]: I don't have good words for a description here. Probably should briefly describe mission and development/stability/supported status.

File: projects/effortless.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Josh O'Brien
+  - Github: [OBrienCommaJosh](https://github.com/OBrienCommaJosh)
+  - Slack: @job
+
+Mark Dennard
+  - Github: [mdennard](https://github.com/mdennard)
+
+#### Project Approvers
+
```
Comment (TheLunaticScripter) [inline projects/effortless.md]: I think we should also add Jon Cowie to reviewers.

File: projects/effortless.md (hunk 1 above)
Comment (TheLunaticScripter) [reply projects/effortless.md]: Yep I agree I'll put some thought into this

File: projects/effortless.md (hunk 2 above)
Comment (robbkidd) [reply projects/effortless.md]: Added.

Comment (robbkidd) [reply projects/effortless.md]: Added.

File: projects/effortless.md (hunk 1 above)
Comment (TheLunaticScripter) [reply projects/effortless.md]: 🤔 "The Effortless project includes the Habitat Scaffolding for Chef and InSpec and the things to make using that easy." Or something like that maybe

**Comment by robbkidd** [comment]
☝️ Also also, the list of humans documented here and the members of [the similarly-named teams in GitHub](https://github.com/orgs/chef/teams/effortless) should match.

File: projects/effortless.md (hunk 1 above)
Comment (robbkidd) [reply projects/effortless.md]:
```suggestion
The Effortless project includes the Habitat Scaffolding for Chef Infra and Chef InSpec and the things to make using them easy.
```

File: projects/effortless.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+David Echols
```
Comment (echohack) [inline projects/effortless.md]: ??? I am a project owner.

File: projects/effortless.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+John Snow
+  - Github: [thelunaticscripter](https://github.com/thelunaticscripter)
+  - Slack: @thelunaticscripter
+
+Josh O'Brien
+  - Github: [OBrienCommaJosh](https://github.com/OBrienCommaJosh)
+  - Slack: @job
+
+Mark Dennard
```
Comment (echohack) [inline projects/effortless.md]: No offense to Mark here, but he does not have a single commit to the repo. How is he consider a project owner?

File: projects/effortless.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# Effortless
+
+The Effortless project includes the Habitat Scaffolding for Chef Infra and Chef InSpec and the things to make using them easy.
```
Comment (echohack) [inline projects/effortless.md]: This is not the words written in the project.

Please replace this with:

`[Effortless](https://github.com/chef/effortless/blob/master/README.md) is automated best practices for Chef Infra and Chef InSpec.`

File: projects/effortless.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+Scott Ford
+  - Github: [scottford-io](https://github.com/scottford-io)
+
+Susan Evans
+  - Github: [susanev](https://github.com/susanev)
+
+#### Project Reviewers
+
+Robb Kidd
```
Comment (echohack) [inline projects/effortless.md]: Robb you are currently assigned an an approver, and I have not received a request to make you a reviewer. Did you want to increase your responsibility on effortless? Or is this a misnomer?

Comment (robbkidd) [reply projects/effortless.md]: Reviewers: have opinions
Approvers: can approve opinions

I do not believe my responsibilities in the project are to approve opinions. My contributions are in the flavor of having opinions for Approvers and Owners to consider.

File: projects/effortless.md (hunk 4 above)
Comment (TheLunaticScripter) [reply projects/effortless.md]: The reason that Mark is listed as a Project Owner is this represents the Product Team from Product and Engineering. That is why the Owner list is this way with Mark as Designer, Josh as Product Manager, and myself as Engineer Lead

File: projects/effortless.md (hunk 5 above)
Comment (TheLunaticScripter) [reply projects/effortless.md]: The Project is more than just what is contained in the Effortless that is why it is different than what is in the Effortless repo.

File: projects/effortless.md (hunk 6 above)
Comment (echohack) [reply projects/effortless.md]: roger, I'll move you to reviewers

**Comment by tas50** [comment]
@robbkidd When we pull the trigger on this make sure to update the CODEOWNERS file with the teams and get those setup in Github

File: projects/effortless.md (hunk 5 above)
Comment (afiune) [inline projects/effortless.md]: I'm not 100% sure this is the right wording we want to use to describe `Effortless`, is it a project? or a combination of patterns?

Comment (kmf) [reply projects/effortless.md]: Workflow?

**Comment by tas50** [comment]
Closing this out since we're not considering effortless its own project at this point.

**Pull request closed without merging**